Protocol-Draft-v3.md

```diff
@@ -265,10 +272,21 @@ Offsets/addresses that start with $ are hexadecimal, otherwise they are decimal.
 
 * If offset is empty, read entire memory.
 * If size is empty, read from offset to end.
+* If offsetN is given, sizeN can not be omitted for N>=2
 * If reading out of bounds for last offset/size: reply can be short (reply has less bytes than requested).
 * If reading out of bounds for non-last offset/size: reply has to be padded with 0.
 * If **all** addresses are out of bounds the reply can be empty instead of sending padding.
-* If offsetN is given, sizeN can not be omitted for N>=2
+
```

Comment:
Again implementation details x)

Reply:
this is not an implementation detail. the command is invalid if the length is missing for reads/writes that have multiple regions